django_project/realtime/tasks/headless/test_headless_task.py

```diff
@@ -0,0 +1,258 @@
+# coding=utf-8
+"""Docstring here."""
+
+import os
+import unittest
+from realtime.tasks.headless.inasafe_wrapper import (
+    get_keywords,
+    run_analysis,
+    generate_contour,
+    run_multi_exposure_analysis,
+    generate_report,
+    get_generated_report,
+    check_broker_connection,
+)
+
+
+common_test_data_path = os.environ.get(
+    'COMMON_TEST_DATA', '/home/headless_test/data/')
+
+dir_path = common_test_data_path
+
+# Layers
+earthquake_layer_uri = os.path.join(
+    dir_path, 'input_layers', 'earthquake.asc')
+shakemap_layer_uri = os.path.join(
+    dir_path, 'input_layers', 'grid-use_ascii.tif')
+place_layer_uri = os.path.join(
+    dir_path, 'input_layers', 'places.geojson')
+aggregation_layer_uri = os.path.join(
+    dir_path, 'input_layers', 'small_grid.geojson')
+population_multi_fields_layer_uri = os.path.join(
+    dir_path, 'input_layers', 'population_multi_fields.geojson')
+buildings_layer_uri = os.path.join(
+    dir_path, 'input_layers', 'buildings.geojson')
+
+# Map template
+custom_map_template_basename = 'custom-inasafe-map-report-landscape'
+custom_map_template = os.path.join(
+    dir_path, custom_map_template_basename + '.qpt'
+)
+
+OUTPUT_DIRECTORY = os.environ.get(
+    'INASAFE_OUTPUT_DIR', '/home/headless_test/data/result/')
+
+
+@unittest.skipIf(
+    os.environ.get('TRAVIS', False), 'Test is not ready for Travis')
```

Comment:
So, it is skipped for now? :p

Reply:
Yeah, we need to setup the inasafe-headless in the travis. I am not sure what's the best way to do it in travis.